src/shoppingList/repository/shoppingListRepository.js

```diff
@@ -0,0 +1,12 @@
+import { prismaClient } from "../../../database/prismaClient.js";
+
+async function createShoppingList(title, userId) {
```

Comment:
Vi em em alguns momentos você cria classes e outras exporta apenas funções.

No seu padrão adotado, repositórios deve realmente ser várias funções soltas e controladores sempre classes ?

Reply:
Então, pros controllers coloquei como padrão Classes. 
Para as outras partes to tentando seguir a ideia do JS funcional, criando funções separadas e exportando elas, o meu controller, como to fazendo um controller por caso de uso to usando classe mesmo, mas poderia mudar pra function e exportar sem problema nenhum,